notebooks/02_twitter_injest.scala

```diff
@@ -24,20 +24,22 @@ val eventhub_namespace = dbutils.preview.secret.get("storage_scope", "eventhub_n
 val eventhub_input = dbutils.preview.secret.get("storage_scope", "eventhub_input")
 val eventhub_key = dbutils.preview.secret.get("storage_scope", "eventhub_key")
 val eventhub_keyname = "RootManageSharedAccessKey"
-val connStr = new ConnectionStringBuilder()
-  .setNamespaceName(eventhub_namespace)
-  .setEventHubName(eventhub_input)
-  .setSasKeyName(eventhub_keyname)
-  .setSasKey(eventhub_key)
+val connStr = s"Endpoint=sb://${eventhub_namespace}.servicebus.windows.net/;" +
```

Comment:
Why all the "s" chars?

Reply:
https://docs.scala-lang.org/overviews/core/string-interpolation.html